### DISCUSSION TITLE
Device Name & Static over not (Windows)

### DISCUSSION THREAD

**Discussion by adamneveu** [philippe44/AirConnect#440] [Q&A]
Hi everyone,

New to this solution but it worked pretty much right away. I am running windows and have it set up with the service auto runner thing which is awesome, worked right away.

My coding and command prompt knowledge isn't what it used to be but, I noticed my device names are all ending in a '+'? Any way to change that?
![image](https://user-images.githubusercontent.com/40398102/205331299-3192735b-b386-48dd-ab15-437ba4df565a.png)

I am using cast-static.exe even though it said not recommended but I couldn't figure out why the regular cast.exe wasn't working on my windows computer.

Is there any downside to using cast-static.exe?

Thank you in advance for any help or answers! 🙏

**Comment by philippe44** [comment]
You can create a config file (see README) and rename the player to whatever you like. Re the static version, which windows do you have? Have you copied the dll as well?

**Comment by adamneveu** [comment]
Wow thank you for such a quick reply.

Ah ok I wasn't sure if it was something that I was missing for static directory, I will make that, and rename them in there so it drops that + symbol.

windows 10, I did copy .dlls to the Windows/system32 folder. It errored saying those were missing, then I added them, and the app still crashed without saying it was missing the .dlls it was strange so I tried static and it worked instantly so that was sweet.

Is there a downside to static?

**Comment by philippe44** [comment]
Static won't let you benefit from mainly openSSL updates. The DLL should be copied int the directory where the app runs. Can you tell me what message you have when you start it?

**Reply by adamneveu** [reply]
Got it - so the error I get its looking for libssl-1_1d.dll

When I add them to Windows32 folder with the rest of the system .dlls it will say 'Application failed to start' with an ok prompt and that's about it. 

Static ran, and I set it as a startup service, and has been working flawless since but would rather have this installed properly of course :)
